sal_interfaces/Rotator/Rotator_Events.xml

```diff
@@ -110,78 +110,54 @@
         <Units>unitless</Units>
         <Count>1</Count>
     </item>
-    <item>
-        <EFDB_Name>timestamp</EFDB_Name>
-        <Description>timestamp</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
 </SALEvent>
 <SALEvent>
     <Subsystem>Rotator</Subsystem>
     <Version>3.5.1</Version>
     <Author></Author>
-    <EFDB_Topic>Rotator_logevent_trackLost</EFDB_Topic>
-    <Alias>trackLost</Alias>
-    <Description>This event is generated if system stops tracking within limits or if we do not receive a new track command within 150ms (three iterations) following the last track command</Description>
+    <EFDB_Topic>Rotator_logevent_target</EFDB_Topic>
+    <Alias>target</Alias>
+    <Description>Commanded target position, velocity and time.</Description>
     <item>
-        <EFDB_Name>timestamp</EFDB_Name>
-        <Description>timestamp</Description>
+        <EFDB_Name>position</EFDB_Name>
+        <Description>Target position at the time specified by "tai".</Description>
         <IDL_Type>double</IDL_Type>
-        <Units>unitless</Units>
+        <Units>deg</Units>
         <Count>1</Count>
     </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_logevent_tracking</EFDB_Topic>
-    <Alias>tracking</Alias>
-    <Description>Tracking is successful and this event is issued whenever once 3 consecutive 50 ms track commands have been applied</Description>
     <item>
-        <EFDB_Name>timestamp</EFDB_Name>
-        <Description>timestamp</Description>
+        <EFDB_Name>velocity</EFDB_Name>
+        <Description>Target velocity at the time specified by "tai".</Description>
         <IDL_Type>double</IDL_Type>
-        <Units>unitless</Units>
+        <Units>deg/second</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>tai</EFDB_Name>
+        <Description>TAI time of the "position" and "velocity" fields.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>second</Units>
         <Count>1</Count>
     </item>
 </SALEvent>
 <SALEvent>
     <Subsystem>Rotator</Subsystem>
     <Version>3.5.1</Version>
     <Author></Author>
-    <EFDB_Topic>Rotator_logevent_deviceError</EFDB_Topic>
```

Comment:
Same as the deviceError event for hexapod. I prefer to keep this if it is not too much trouble.

Reply:
I'm afraid it's not going to happen, for the reasons given above.